.circleci/config.yml

```diff
@@ -98,6 +101,32 @@ commands:
 #    - store_artifacts:
 #        path: ./build/libs
 
+  release2pypi:
+    steps:
+      - run:
+          name: Reset PyPI credentials
+          command: |
+            if [ -z "$CIRCLE_PULL_REQUEST" ]; then
+               echo "Reset PyPI Password"
+               echo "export TWINE_PASSWORD=$PYPI_PASSWORD" >> "$BASH_ENV"
```

Comment:
is this needed ? why not use the secret direct from the circle secret store ? or maybe i am missing something here

Reply:
As explained, the "built in" password and username for twine are TWINE_PASSWORD and TWINE_USERNAME. These are defined in the project settings. USERNAME is the same for both testPyPI and PyPI, however the PASSWORD differs. In case of a release to PyPI (i.e. when the PR is merged) this therefore needs to be reset.